src/main/scala/com/exasol/spark/util/Filters.scala

```diff
@@ -51,9 +51,11 @@ object Filters extends LazyLogging {
       case _                        => None
     }
 
-  def isStringValue(dataType: DataType, value: Any): String = dataType match {
-    case StringType => s"'$value'"
-    case _          => s"$value"
+  def isQuotedValue(dataType: DataType, value: Any): String = dataType match {
+    case StringType    => s"'$value'"
+    case DateType      => s"date '$value'"
+    case TimestampType => s"date '$value'"
```

Comment:
Should this be `timestamp` instead of `date`?

Reply:
Yes, I thought that syntax is not available for timestamp. It might be my error why double checking it in EXAplus.